packages/plutolang/build.sh

```diff
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
 TERMUX_PKG_DESCRIPTION="A superset of Lua 5.4, with unique features, optimizations, and improvements"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Would you mind setting yourself as the maintainer for this package?

Reply:
Resolved in [9b2b93e](https://github.com/termux/termux-packages/pull/19695/commits/9b2b93eb2ed0593d34253e841ad0f74c6db51972).

Reply:
Will still need to be squashed.
We *can* do that during Merge, but usually we prefer the author to do this in order to ensure everything is being committed as intended.

You can squash the commits using `git rebase -i HEAD~2`, and selecting `s`quash for the second commit to merge it into the first one.

Reply:
The commits are now squashed.

Reply:
Thank you.
Is this ready to merge then, or are you still testing it?

Reply:
I still need to test `-fstack-protector-strong` on the Pluto `testes/` directory, I'll inform you once I'm done.

Reply:
Done tested it, you may merge this now.